src/views/MountainLayout/MountainLayoutViewData.ts

```diff
@@ -6,14 +6,16 @@ import { isString } from "vega"
 type MLViewData = {
     label: string
     type: string
-    figureDataSha1: Sha1Hash
+    figureDataSha1?: Sha1Hash // old
+    figureDataUri?: string // new
 }
 
 const isMLViewData = (x: any): x is MLViewData => {
     return validateObject(x, {
         label: isString,
         type: isString,
-        figureDataSha1: isSha1Hash,
+        figureDataSha1: optional(isSha1Hash), // old
+        figureDataUri: optional(isString) // new
```

Comment:
Same note as with `CompositeViewData`, is it possible to require that exactly one of these is set for the type guard to pass? I noticed it has to be checked in a couple of different consumers otherwise.